meteor-app/imports/ui/assessment/assessment-add.js

```diff
@@ -108,11 +108,14 @@ class AssessmentAdd extends Component {
         .map(key => {
           return {
             name: key.name, 
-            price: key.price
+            price: key.price,
+            code: key.code,
+            category: key.category,
+            used: key.used
           }
         })
       const custName = formData.b2bRefurbish ? 'back2bikes' : formData.name.toLowerCase()
-      const search = custName + formData.email + formData.bikeMake.toLowerCase() + formData.bikeColor.toLowerCase()
+      const search = custName + formData.email + formData.phone + formData.bikeMake + formData.bikeModel + formData.bikeColor + formData.mechanic + formData.assessor + formData.comments + formData.package + JSON.stringify(partsItem)
```

Comment:
This code works fine, but there is a way that we can future proof your code like this.
```
const search = Object.values(formData).join(" ")
```
So if and when (which is likely) we add more information to the form, we won't need to change this code again.